code/go/0chain.net/chaincore/chain/config.go

```diff
@@ -412,8 +426,11 @@ type ConfigData struct {
 	RoundTimeoutSofttoMult int `json:"softto_mult"`        // multiplier of mean network time for soft timeout
 	RoundRestartMult       int `json:"round_restart_mult"` // multiplier of soft timeouts to restart a round
 
-	DbsEvents config.DbAccess `json:"dbs_event"`
-	TxnExempt map[string]bool `json:"txn_exempt"`
+	DbsEvents   config.DbAccess   `json:"dbs_event"`
+	DbsSettings config.DbSettings `json:"dbs_settings"`
+	TxnExempt   map[string]bool   `json:"txn_exempt"`
+
+	LastUpdateRound int64
```

Comment:
I'm thinking about if this LastUpdateRound is needed, we will update configs only when finalizing blocks, and basically most of the events are processed when finalizing blocks, so the config should already have been updated then.